### DISCUSSION TITLE
Make create team on user creation optional and automatically add new users to teams they are invited for in JetStream

### DISCUSSION THREAD

**Discussion by seebaermichi** [laravel/framework#49703] [Ideas]
Hi,
I'm Michael, a freelancer from Germany. I'm working with Laravel for about six years now. But I used Laravel JetStream today for the first time. It's a really nice package and I especially like the team feature.

While implementing and testing everything I figured out, that for each new user, there is always a team created. Why is this? It felt really weird to me, because in my opinion there is no need, especially when I invite a new team member, that this team member has then her own team and the one she was invited to.

So I would like to make a proposal. The feature to create always a new team, if a new user is created should be configurable. It could stay as the default, but there also should be the option, that this won't be the case. And a new user can just have no team initially.
In addition new users are added automatically to a team, if there is an invite with their email.

Since I already adjusted the code for my current project. I would love to create a pull request for the described changes, if this is something you would like to have in the package.

What do you think? Maybe I miss something and there are good reasons to not have the options I described above. Looking forward to your thoughts.

**Comment by acharseth** [comment]
I too wonder if it is necessary to create a default team for new users.
I also wonder how to skip the email invite and instead have functionality to directly add a user to a team. Is there any reason why this should not be possible?
